src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -410,6 +410,7 @@ const Inventory = ({
             isDisabled={isRemediationButtonDisabled}
             dataProvider={remediationDataProvider}
             onRemediationCreated={(result) => onRemediationCreated(result)}
+            hasSelected={selectedIds.length > 0}
```

Comment:
<img width="2504" height="1317" alt="Image" src="https://github.com/user-attachments/assets/c969e904-51ea-4d62-ab8f-00bdef4041fe" />


```suggestion
            hasSelected={selectedIds?.length > 0}
```